packages/fiori-elements-writer/test/common.ts

```diff
@@ -5,17 +5,21 @@ import { create as createStore } from 'mem-fs';
 import { create } from 'mem-fs-editor';
 import { join } from 'path';
 import type { FEOPSettings, FioriElementsApp, LROPSettings, WorklistSettings } from '../src/types';
+import { promisify } from 'util';
+import { exec as execCP } from 'child_process';
+const exec = promisify(execCP);
 
 export const testOutputDir = join(__dirname, 'test-output');
 
 export const debug = prepareDebug();
 
-export function prepareDebug(): { enabled: boolean; outputDir: string } {
+export function prepareDebug(): { enabled: boolean; outputDir: string; debugFull: boolean } {
     const debug = !!process.env['UX_DEBUG'];
+    const debugFull = !!process.env['UX_DEBUG_FULL'];
```

Comment:
Can we add some documentation about the use of these 2 debug env vars, since the naming is a little obscure, please.

Reply:
Actually we already have docs for `UX_DEBUG` so maybe we can add `UX_DEBUG_FULL` also : https://github.com/SAP/open-ux-tools/blob/9d4e3bcfd4a1d45e93ba603ecf345e34896cedf8/README.md

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/914/commits/26fac5bc8aef6ebcbb2ba993fe3ccbd9431780d1